l1-contracts/test/foundry/unit/concrete/AddressAliasHelper/actualRefundRecipient.t.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {AddressAliasHelperSharedTest} from "./_AddressAliasHelper_Shared.t.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract TestContract is Test {
+    function func() public pure returns (bool) {
```

Comment:
Why?

Reply:
I needed a minimal contract to check if the ```refundRecipient``` is a contract.

Reply:
I simplified this contract and added comment